package.json

```diff
@@ -110,5 +110,8 @@
     "vite": "^4.4.5",
     "vite-plugin-dts": "^3.5.2",
     "vite-plugin-eslint": "^1.8.1"
+  },
+  "dependencies": {
+    "classnames": "^2.3.2"
```

Comment:
let's remove this dependency for now and explore this pkg and refactoring components outside of the scope of ads-712.